research-software-directory/RSD-as-a-service
# Draft: Auto fill licenses from DOI

# Auto fill licenses from DOI

Fixes https://github.com/research-software-directory/Helmholtz/issues/9

Changes proposed in this pull request:

* fetch licenses from DOI information (using DataCite API)

How to test:

* create a software entry and fill in a DOI containing license information, click on "Import licenses", the licenses from the DOI should then be added
* using a DOI without license information should not add licenses and show a message instead

PR Checklist:
